<launch>
    <arg name="serial_port" default="/dev/tty/ACM1"/>

    <node pkg="rosserial_arduino" type="serial_node.py" name="hardware" args="$(arg serial_port)">
        <param name="THROTTLE_MAX_FORWARD" value="2000"/>
        <param name="THROTTLE_ZERO" value="1500"/>
        <param name="THROTTLE_MAX_BACKWARD" value="1000"/>
        <param name="STEERING_MAX_RIGHT" value="2000"/>
        <param name="STEERING_ZERO" value="1500"/>
        <param name="STEERING_TRIM" value="0"/>
        <param name="STEERING_MAX_LEFT" value="1000"/>
    </node>
</launch>
